### PR TITLE
add layout product card

### PR DESCRIPTION
**Pull request by VvynnykV**
- [DEMO LINK](https://VvynnykV.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://VvynnykV.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/styles/stars.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
```
Comment (VitaliyBondarenko1982) [inline src/styles/stars.css:8]: do not set margin for last star

**Review by VitaliyBondarenko1982 — CHANGES_REQUESTED** [review]
hover looks like to work correctly
try to remove backstop_data and run `npm test` again locally

**Comment by VvynnykV** [comment]
> hover looks like to work correctly try to remove backstop_data and run `npm test` again locally

it helps)

File: src/styles/stars.css (hunk 1 above)
Comment (VvynnykV) [reply src/styles/stars.css:8]: done

**Comment by VvynnykV** [comment]
thanks for your support)

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,54 @@
...
+        <p class="card__price-item">Price:</p>
+        <p class="card__price-amount">$2,199</p>
+      </div>
+
+      <a class="card__button"
+      href="./"
+      data-qa="hover"
+      >
+        Buy
+      </a>
```
Comment (IvanFesenko) [inline src/index.html:51]: Pay attention on indents
```suggestion
      <a 
        class="card__button"
        href="./"
        data-qa="hover"
      >
        Buy
      </a>
```

File: src/styles/stars.css [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background: no-repeat center url(../images/star-active.svg);
+  /* background-image: url(../images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center; */
```
Comment (IvanFesenko) [inline src/styles/stars.css:11]: Remove commented code

File: src/styles/card.css [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+  text-transform: uppercase;
+  color: #FFF;
+  text-decoration: none;
+  padding: 11px 0;
+}
+
+.card__button:hover {
+  background: #FFF;
+  border: 1px solid #00ACDC;
+  color: #00ACDC;
```
Comment (IvanFesenko) [inline src/styles/card.css:89]: Move repatable color to variable and use it

**Review by IvanFesenko — CHANGES_REQUESTED** [review]
Some small changes required

**Pull request closed without merging**